packages/next/client/index.js

```diff
@@ -301,7 +301,16 @@ function renderReactElement(reactEl, domEl) {
   }
 
   if (onPerfEntry) {
-    performance.getEntriesByType('paint').forEach(onPerfEntry)
+    if (!(PerformanceObserver in window)) {
+      window.addEventListener('load', () => {
+        performance.getEntriesByType('paint').forEach(onPerfEntry)
```

Comment:
Will this throw in certain older browsers?

Reply:
Yep makes sense to do feature detection for window.performance

Reply:
like `performance && performance.getEntriesByType('paint').forEach(onPerfEntry)`?

Asking because i see the whole file using `performance`. should we add it everywhere?

Reply:
@prateekbh I was referencing this: https://github.com/zeit/next.js/blob/canary/packages/next/client/index.js#L309

Reply:
done